src/components/layout/layout.js

```diff
@@ -34,6 +34,12 @@ export default function Layout({ children }) {
           <p>About Our Organ Donation System</p>
         </Link>
       </Row>
+      {crumbLabel ? (
+        <Breadcrumb>
+          <Breadcrumb.Item href="/">Home</Breadcrumb.Item>
```

Comment:
It doesn't look like the linking is working? Do we need to actually add a `Link` component within the breadcrumb, do you think?

Reply:
That is strange, the Home link works to redirect the route back to the index for me when I click it. Is that what you mean?

Reply:
Oh interesting I must have misclicked--ignore me then, this works!